jsapp/js/account/organization/membersQuery.ts

```diff
@@ -34,6 +47,62 @@ export interface OrganizationMember {
   };
 }
 
+function getMemberEndpoint(orgId: string, username: string) {
+  return endpoints.ORGANIZATION_MEMBER_URL
+    .replace(':organization_id', orgId)
+    .replace(':username', username);
+}
+
+/**
+ * Mutation hook for updating organization member. It ensures that all related
+ * queries refetch data (are invalidated).
+ */
+export function usePatchOrganizationMember(username: string) {
```

Comment:
In the hypothetical situation where we could need to call multiple `usePatch...` in the same view we might want to have the username as a parameter of the `mutationFn` itself instead of the hook.

But that's only to stay in our minds for the future. We can change this is needed.